Mogakco/Sources/Domain/Entities/EmailLogin.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  EmailLogin.swift
+//  Mogakco
+//
+//  Created by 이주훈 on 2022/11/21.
+//  Copyright © 2022 Mogakco. All rights reserved.
+//
+
+import Foundation
+
+struct EmailLoginData {
+    let email: String
+    let password: String
+}
```

Comment:
```suggestion
import Foundation

struct EmailLogin {
    let email: String
    let password: String
}
```
Data로 접미어를 사용하게 되면 자료형이 Data인가하는 헷갈릴 수 있을 것 같아요!